src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx

```diff
@@ -34,13 +42,23 @@ const FormFieldErrorReason = styled.div`
 const FormFieldError = styled.div`
   color: ${p => p.theme.redDark};
   animation: ${pulse} 1s ease infinite;
-  width: 16px;
-  margin-left: auto;
 `;
 
 const FormFieldIsSaved = styled.div`
   color: ${p => p.theme.green};
   animation: ${fadeOut} 0.3s ease 2s 1 forwards;
+  position: absolute;
```

Comment:
Kind of hacky, but would have weird behavior if "saved" happens followed by "error" where both will be rendered

(current prod)
![image](https://user-images.githubusercontent.com/79684/34591765-a9dbbe76-f174-11e7-918a-2206df02b91a.png)